cmd/start.go

```diff
@@ -91,3 +110,98 @@ func trapSignal(done func()) {
 	// call the cleanup func
 	done()
 }
+
+// UpdateClientsFromChains takes src, dst chains and update clients based on expiry time
+func UpdateClientsFromChains(src, dst *relayer.Chain) (err error) {
+	if src.PathEnd.ClientID == "" || dst.PathEnd.ClientID == "" {
+		return nil
+	}
+	eg := new(errgroup.Group)
+	eg.Go(func() error {
+		err = GetClientAndUpdate(src, dst)
+		return err
+	})
+	eg.Go(func() error {
+		err = GetClientAndUpdate(dst, src)
+		return err
+	})
+	err = eg.Wait()
+	return err
+}
+
+// GetClientAndUpdate update clients to prevent expiry
+func GetClientAndUpdate(src, dst *relayer.Chain) error {
```

Comment:
This logic can be moved into `relayer/`. I think we should actually just not check if the client is expired. If the time interval has passed, we should attempt to update the client using `src.UpdateClient(dstUH),` 

The benefit of checking if the client is not spending gas on a message bound to fail. Given that relayers will burn a lot more gas just updating the client, I'm not too worried about this and would prefer to reduce the amount of code added

thoughts @fedekunze @AdityaSripal

Reply:
It makes sense to check. Otherwise, we'll keep sending update messages to an expired client every minute until someone manually intervenes and stops the relayer from doing so.

Reply:
@colin-axner, if we don't want to check expiry, then this function is not needed. We can just call `src.UpdateClients(dst)` in go routine with time interval.